# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/nsPresContext.cpp
# Commit: 3d30a8d83d77
# Full Hash: 3d30a8d83d77cfb25c9d75aae2daa01051810ff0
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771493
# File Overlap Count: 1
# Description:
#   Bug 1771493 - Part 1. Split FontFace(Set) into DOM facing and implementation classes. r=emilio
#   
#   This patch aims to split the implementation details of FontFace(Set)
#   into FontFace(Set)Impl classes. The Impl variants have threadsafe
#   refcounting and are intended to be the basis for worker support for
# ==============================================================================

diff -r 53c490e52eb6 -r 3d30a8d83d77 layout/base/nsPresContext.cpp
--- a/layout/base/nsPresContext.cpp	Tue Jun 28 21:23:39 2022 +0000
+++ b/layout/base/nsPresContext.cpp	Tue Jun 28 21:46:59 2022 +0000
@@ -65,6 +65,7 @@
 #include "mozilla/dom/PBrowserParent.h"
 #include "mozilla/dom/BrowserChild.h"
 #include "mozilla/dom/BrowserParent.h"
+#include "mozilla/dom/FontFaceSet.h"
 #include "mozilla/StaticPresData.h"
 #include "nsRefreshDriver.h"
 #include "Layers.h"